spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -0,0 +1,497 @@
+import sys
+import os
+import numpy as np
+import csv
+import logging
+from typing import Sequence
+import pandas as pd
+from spinalcordtoolbox.utils import SCTArgumentParser, Metavar, init_sct, set_loglevel
+from spinalcordtoolbox.utils.fs import get_absolute_path, check_file_exist, extract_fname, printv
+from spinalcordtoolbox.image import Image
+from spinalcordtoolbox import __data_dir__
+
+
+logger = logging.getLogger(__name__)
+
+
+# PARSER
+# ==========================================================================================
+def get_parser():
+    parser = SCTArgumentParser(
+        description='Compute spinal cord compression morphometrics such as the Maximum Spinal Cord Compression [1], '
+                    'the antero-posterior (AP) diameter, or other relevant spinal cord morphometrics that are output'
+                    ' by the function sct_process_segmentation (CSA, RL diameter, eccentricity, solidity, etc.).'
+                    ' Metrics are normalized using a database of spinal cord morphometrics built from healthy control'
+                    ' subjects. This database uses the PAM50 template as an anatomical reference system.'
+                    ' \nEquation:    ratio = (1 - mi/((ma+mb)/2))\n'
+                    'mi: metric at compression level. ma: metric of level above compression level. mb: metric of level below compression level.'
+                    '[1]: Miyanji F, Furlan JC, Aarabi B, Arnold PM, Fehlings MG. Acute cervical traumatic spinal cord injury:'
+                    ' MR imaging findings correlated with neurologic outcome--prospective study with 100 consecutive'
+                    ' patients. Radiology 2007;243(3):820-827.'
+    )
+
+    mandatoryArguments = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatoryArguments.add_argument(
+        '-i',
+        metavar=Metavar.file,
+        required=True,
+        help='CSV morphometric file obtained with sct_process_segmentation. Example: csa.csv.'
+    )
+    mandatoryArguments.add_argument(
+        '-l',
+        metavar=Metavar.file,
+        required=True,
+        help='NIfTI file that includes labels at the compression sites. Each compression site is denoted by a single voxel of value `1`. '
+             'Example: sub-001_T2w_compression_labels.nii.gz'
+    )
+    mandatoryArguments.add_argument(
+        '-metric',
+        required=False,
+        help='Metric to normalize.'
+        'Choices: area, diameter_AP, diameter_RL, eccentricity, solidity\n',
+        default='diameter_AP',
+        choices=['diameter_AP', 'area', 'diameter_RL', 'eccentricity', 'solidity'],
+        metavar=Metavar.file,
+    )
+    optional = parser.add_argument_group("\nOPTIONAL ARGUMENTS")
+    optional.add_argument(
+        '-i-PAM50',
+        metavar=Metavar.file,
+        help='CSV morphometric file in the PAM50 space, obtained by running: '
+        'sct_process_segmentation -normalize-PAM50.'
```

Comment:
Maybe we could mention that it is necessary to provide this file if one wants to do normalization using the database of healthy control. 

```suggestion
        help='CSV morphometric file in the PAM50 space, obtained by running: '
        'sct_process_segmentation -normalize-PAM50. Use this flag if you want to normalize the metrics using a database of healthy controls.'
```